pkg/infra/baseagent.go

```diff
@@ -49,6 +49,14 @@ func (a *Base) HP() int {
 	return a.hp
 }
 
+func (a *Base) CurrPlatFood() float64 {
+	foodOnPlatform, platformFloor := a.tower.GetCurrPlatFood()
+	if platformFloor == a.floor { // only show the food on the platform if the platform is on the same floor as the agent
```

Comment:
Or a.floor-1 (possibly):

Since we can communicate with the agent below, I assume we can see the food at this point too. At the same time, I can't see how this is immediately useful (potentially opinion formation regarding lying)